src/main/java/net/mcreator/ui/views/editor/image/ImageMakerView.java

```diff
@@ -354,7 +354,7 @@ public static boolean isFileSupported(String fileName) {
 		MCreatorTabs.Tab existing = mcreator.mcreatorTabs.showTabOrGetExisting(this.tab);
 		if (existing == null) {
 			mcreator.mcreatorTabs.addTab(this.tab);
-			leftSplitPane.setDividerLocation(1.2 / 8);
+			leftSplitPane.setDividerLocation(1.3 / 7);

```

Comment:
On my screen, this makes the tool panel too wide:

![image](https://github.com/MCreator/MCreator/assets/16374228/7a9f1dd4-e943-4064-a14d-723cea9c8bc2)

Compared to how it looks on the master branch on my screen (looks better):

![image](https://github.com/MCreator/MCreator/assets/16374228/c34309ea-c97e-45b6-952b-f68e2f00fe97)

Seems every parameter works better for different screens, so we need to find a universal solution, not adjust to work best on just one screen.